test/metabase/upload_test.clj

```diff
@@ -1511,6 +1510,41 @@
                    (rows-for-table table))))
           (io/delete-file file))))))
 
+(deftest csv-append-snowplow-test
+  ;; Just test with h2 because snowplow should be independent of the driver
+  (mt/test-driver :h2
+    (snowplow-test/with-fake-snowplow-collector
+
+     (with-upload-table! [table (create-upload-table!)]
+       (let [csv-rows ["name" "Luke Skywalker"]
+             file     (csv-file-with csv-rows (mt/random-name))]
+         (append-csv! {:file file, :table-id (:id table)})
+
+         (is (=? {:data {"size_mb"         1.811981201171875E-5
```

Comment:
```suggestion
         (is (=? {:data {"size_mb"         (mt/malli=? pos-int?)
```
I'd avoid testing the exact size here

Reply:
Fair enough, there could be some platform sensitivity. What do you think about matching the order of magnitude eat least?

Reply:
I've decided to leave this using the exact size:

1. Our initial upload case uses an explicit number.
2. We don't really have any other tests of the size calculation helper.
3. It seems to be deterministic at least on unix platforms.
4. It's easy to update if the value changes.